packetbeat/etc/beat.yml

```diff
@@ -17,6 +17,10 @@ interfaces:
 
 ############################# Protocols #######################################
 protocols:
+  icmp:
+    # Enable ICMPv4 and ICMPv6 monitoring. Default: false
+    enabled: true
```

Comment:
Don't we normally have in the config the default value? So it should be false here?